docs/swagger.yaml

```diff
@@ -12,17 +12,17 @@ schemes:
   - https
 
 securityDefinitions:
-  api_key:
-    type: apiKey
-    description: API password
-    name: api_password
-    in: query
-  
-  # api_key:
+  #api_key:
```

Comment:
I concur, which is why I commented out the query definition.  If someone were to need it for some reason, they could switch back.